community/neo4j/src/test/java/schema/MultipleIndexPopulationStressIT.java

```diff
@@ -103,6 +97,7 @@
     private final TestDirectory directory = TestDirectory.testDirectory();
 
     private final RandomRule random = new RandomRule();
+    private RandomValues randomValues;
```

Comment:
This looks like an annoyance, I want RandomRule to play nice with RandomValues, not be its competitor... can it just provide it seamlessly? It should be enough to have your RandomRule in your test and from there you can get everything.